app/assets/stylesheets/screen.scss

```diff
@@ -87,26 +88,94 @@ ul#primary-nav li {
 }
 
 #title_bar {
-  background: #E9E9E9;
-  border-bottom: 1px solid #d5d5d5;
+  background: $lightgrey-1;
   padding: 5px 10px 3px 10px;
   font-size: 1.0em;
   line-height: 140%;
   margin-bottom: 0.5em;
   color: #5e6469;
   position: relative;
   margin: -20px 0 0;
-  padding: 10px 1em;
+  padding: 5px 5px 0px 5px;
+
+  .project-title {
+    text-decoration: none;
+    font-weight: 600;
+    color: $darkgrey-2;
+    transition: 0.3s ease color;
+
+    &:hover {
+      color: $darkred-1;
+    }
+  }
+
+  .right-menu {
+    float: right;
+
+    > .dropdown {
+      display: inline;
+    }
+
+    button {
+      text-decoration: none;
+      font-weight: 600;
+      color: $darkgrey-2;
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    #form_search, .form_locale {
+      display: inline;
+
+      select {
+        color: $darkgrey-2;
+        font-weight: 600;
+        font-size: 14px;
+      }
+
+      .form-control {
+        width: auto;
+        display: inline;
+      }
+
+      .btn-group {
+        width: auto;
+
+        button {
+          background: transparent;
+          border: none;
+          box-shadow: none;
+          width: auto;
+
+          &:active {
+            box-shadow: none;
+          }
+
+          &:focus, span:focus {
+            outline: none;
+          }
+        }
+      }
+    }
+
+    .locale-change {
+      border: 0;
+      box-shadow: none;
+      background-color: transparent;
+    }
+  }
 
   div.velocity {
     font-size: 1.2em;
-    float: right;
+    display: inline;
```

Comment:
I use to follow mdo's code guide to order my properties declarations: http://codeguide.co/#css-declaration-order